Makefile

```diff
@@ -1,10 +1,20 @@
 PACKAGES=$(shell go list ./... | grep -v 'tests')
-HERUMI= $(shell pwd)/.herumi
-BLS_CGO_LDFLAGS=CGO_LDFLAGS="-L$(HERUMI)/bls/lib -lbls384_256 -lm -lstdc++"
-BUILD_LDFLAGS= -ldflags "-X github.com/zarbchain/zarb-go/version.build=`git rev-parse --short=8 HEAD`"
 
+TAGS="zarb"
+VTAG=$(shell git describe --tags --abbrev=0)
+HERUMI= .herumi
+LDFLAGS= -ldflags "-X github.com/zarbchain/zarb-go/version.build=`git rev-parse --short=8 HEAD` -X github.com/zarbchain/zarb-go/version.semVer=$(VTAG)"
```

Comment:
This is really confusing. See my comments about SemVer package down.

Reply:
i might need to add comments to clarify it
but it utilizes git tag command to dynamically set build version to latest tag has been set